src/components/views/rooms/EventTile.tsx

```diff
@@ -1192,14 +1192,17 @@ export default class EventTile extends React.Component<IProps, IState> {
             }
 
             default: {
-                const thread = ReplyThread.makeThread(
-                    this.props.mxEvent,
-                    this.props.onHeightChanged,
-                    this.props.permalinkCreator,
-                    this.replyThread,
-                    this.props.layout,
-                    this.props.alwaysShowTimestamps || this.state.hover,
-                );
+                let thread;
+                if (haveTileForEvent(this.props.mxEvent)) {
```

Comment:
A comment explaining why we're doing this would be appreciated, just so someone doesn't undo the code when they think "well, it'll always have a tile so let's remove this"